iib/web/s3_utils.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import logging
+
+import boto3
+
+log = logging.getLogger(__name__)
+
+
+def get_object_from_s3_bucket(key_prefix, file_key, bucket_name):
+    """
+    Get object from AWS S3 bucket.
+
+    :param str key_prefix: the logical location of the file in the S3 bucket
+    :param str file_key: the name of the file in S3 bucket
+    :param str bucket_name: the name of the S3 bucket to fetch the file from
+    :return: the body of the S3 object or None
+    :rtype: botocore.response.StreamingBody
+    """
+    file_name = f'{key_prefix}/{file_key}'
+    try:
+        log.info('getting file from s3 : %s', file_name)
+        s3 = boto3.resource(service_name='s3')
+        response = s3.meta.client.get_object(Bucket=bucket_name, Key=file_name)
+    except Exception as error:
+        log.debug('Unable to fetch object %s from bucket %s: %s', file_name, bucket_name, error)
```

Comment:
Is `debug` the right level here? This looks like it should be `exception` level.